.gitignore

```diff
@@ -34,3 +34,4 @@ cloud-init_*.upload
 
 # user test settings
 tests/integration_tests/user_settings.py
+misc/
```

Comment:
Let's remove this line since it isn't really relevant for other people. If you want to keep things locally from version control, you can always add the file/directory to `.git/info/exclude`

Reply:
Done. I misread your comment initially, and removed the wrong line. Should be fine now.